src/TLuaInterpreter.cpp

```diff
@@ -2710,11 +2659,11 @@ int TLuaInterpreter::setExitStub(lua_State* L)
         lua_pushstring(L, "setExitStub: RoomId doesn't exist");
         return lua_error(L);
     }
-    if (dirType > 12 || dirType < 1) {
-        lua_pushstring(L, "setExitStub: dirType must be between 1 and 12");
+    if (dir > 12 || dir < 1) {
+        lua_pushstring(L, "setExitStub: direction must be between 1 and 12");
```

Comment:
If you are not going to address this other problem then we'll need a new issue along the lines of:
"Some lua functions that take a (normal) exit as a direction number or a string return confusing error messages for invalid exit strings"
i.e. give the exit direction as "downwards" and the function will abort with a message
"xxxx: direction must be between 1 and 12" - even though a string rather than a number was given.